src/decoder/zlib.rs

```diff
@@ -1,27 +1,22 @@
-use super::{stream::FormatErrorInner, DecodingError, CHUNK_BUFFER_SIZE};
+use super::{stream::FormatErrorInner, DecodingError};
 
 use fdeflate::Decompressor;
 
+pub struct UnfilterBuf<'data> {
+    // The data to be fed into the decoder for (zlib) decompression.
```

Comment:
nit: This vec contains already uncompressed data + space for more data to be uncompressed into